config.h.in

```diff
@@ -32,6 +32,7 @@
 #cmakedefine HAVE_TM_GMTOFF
 #cmakedefine HAVE_AIO_H
 #cmakedefine HAVE_POLL_H
+#cmakedefine HAVE_SYSLOG_H
```

Comment:
@hardening Probably better to define this via compiler argument. Otherwise you'll have to include ```config.h``` in ```wlog.h``` too.